eng/targets/Settings.props

```diff
@@ -21,6 +21,9 @@
 
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
 
+    <!-- This branch builds against net6.0 and net7.0 frameworks. -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
Moving to a newer SDK means new warnings such as EOL check for net6.0 and net7.0. Disabling that check because we know we need to build with those TFMs